recipes/2-Workflows/feature_importance/feature_importance.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "69d7ac54-9a3b-4327-93a5-eb6fbf314530",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Engine Feature Importance\n",
+    "\n",
+    "## Overview \n",
+    " \n",
```

Comment:
Do we want this section? I have been advocating removal within the online docs.

Reply:
yes, this will allow for more information